src/components/top_bar.rs

```diff
@@ -203,6 +203,33 @@ impl TopBar {
 
         ui.separator();
 
+        #[cfg(not(target_arch = "wasm32"))]
+        ui.add_enabled_ui(info.filesystem.project_loaded(), |ui| {
+            if ui.button("Playtest").clicked() {
+                match std::process::Command::new(
+                    info.filesystem.project_path().unwrap().join("steamshim"),
+                )
+                .stdin(std::process::Stdio::piped())
```

Comment:
Please, avoid long paths for structures, enumerations and methods if you're using several of them from one module.
This is purely for the sake of better looking code.

```rust
use std::process;

match process::Command::new(...);
.stdin(process::Stdio::piped())
.stdout(process::Stdio::piped())
```
or
```rust
use std::process::{Stdio, Command};

Command::new(...);
Stdio::piped();
```

Reply:
This uhh isn't the code in the latest commit

Reply:
Iii didn't notice, my apologies-